60th episode (#1104)
quarkusio/quarkusio.github.io: _posts/2021-08-20-60th-quarkus-insights.adoc

---
layout: post
title: Season 3 and 60th Insights episode
date: 2021-08-20   
tags: insights

> synopsis: Quarkus Insights Sesaon 3 starting!

Reviewer: ```suggestion
synopsis: Quarkus Insights Season 3 starting!
```

author: maxandersen
---

:imagesdir: /assets/images/posts

> After a summuer break and little bit of CODVID-19 delay, we will finally have the 60th(!) Quarkus Insights episode on Monday the 23rd August.

Reviewer: ```suggestion
After a summer break and little bit of COVID-19 delay, we will finally have the 60th(!) Quarkus Insights episode on Monday the 23rd August.
```

image:60th.png[]

> For those who don't know Quarkus Insights is a (almost) weekly video/audio podcast where we host people from all parts of the Quarkiverse to sit down and talk about how to use, what is coming next, user stories from the actual users and more!

Reviewer: ```suggestion
For those who don't know, Quarkus Insights is a (almost) weekly video/audio podcast where we host people from all parts of the Quarkiverse to sit down and talk about how to use Quarkus, what is coming next, user stories from the actual users and more!
```

> This monday is titled "The Rise of Platforms" where Alexey will come by and talk about the improvements and changes we've done around the notion of Quarkus Platforms and Extensions for Quarkus 2.x.

Reviewer: ```suggestion
This Monday is titled "The Rise of Platforms" where Alexey will come by and talk about the improvements and changes we've done around the notion of Quarkus Platforms and Extensions for Quarkus 2.x.
```

We have a whole backlog of 60 episodes which for large part are still as relevant as they were when we recorded them.

> You can browse and see the full list at http://quarkus.io/insights/

Reviewer: ```suggestion
You can browse and see the full list at http://quarkus.io/insights/.
```
